content/docs/reference/reference.json

```diff
@@ -104,28 +104,12 @@
   },
   "certificates": {
     "id": "certificates",
-    "title": "Certificates",
+    "title": "Certificates Settings",
     "path": "/certificates",
     "type": "string",
     "short_description": "Certificate relative file location string",
     "services": []
   },
-  "client-certificate-authority": {
-    "id": "client-certificate-authority",
-    "title": "Client Certificate Authority",
-    "path": "/client-certificate-authority",
```

Comment:
Instead of deleting the individual settings / config options, I think it makes sense to keep the entries and link to the consolidated path (e.g. so you can still search for the individual thing).